Migrate SQL docs (#521)
redpanda-data/cloud-docs: modules/reference/pages/sql/sql-functions/math-functions/ln.adoc

= LN
:description: LN() will return the exponential value of its argument, which is recognized as the input parameter’s natural logarithm.
:page-topic-type: reference

`LN()` will return the exponential value of its argument, which is recognized as the input parameter’s natural logarithm.

[NOTE]
====
*Info:*
 The logarithm doesn’t take negative numbers or 0.
====

== Syntax

> The syntax of the `LN()` function is described as follows.

Reviewer: ```suggestion
Syntax:
```

[source,sql]
----
LN (x)
----

`x`:  A positive or a negative number (or an expression that evaluates to a number).

== Examples

=== Basic LN() function

> The following example shows that `LN()` function returns the natural logarithm of the number *7,87653*.

Reviewer: ```suggestion
This example shows how the `LN()` function returns the natural logarithm of the number *7,87653*.
```

[source,sql]
----
SELECT LN(7.87653);
----

The final result is as follows.

[source,sql]
----
+-------------+
| f           |
+-------------+
| 2.0638874   |
+-------------+
----

=== Use LN() function with a table

The following example combines the `LN()` function with a `CREATE TABLE` statement to obtain natural logarithmic values of a specific column.

. Create a new table named *LNTable* containing the *initValue* column with an integer value.
+
[source,sql]
----
CREATE TABLE LNtable(initValue int);
INSERT INTO LNtable(initValue)
VALUES (75), (18), (28);
----

> . Run the following query to get the logarithm output of the column:

Reviewer: ```suggestion
. Run this query to get the logarithm output of the column:
```
